game.cpp

```diff
@@ -43,17 +43,17 @@ void Game::Init() {
     IRender* renderer = GetRenderer();
     renderer->RegisterFont(m_ConsoleFont);
     renderer->RegisterFont(m_ConsoleFont30);
-    
+
     // Load world triangles from Quake .MAP file
 
     std::vector<MapTri> worldTris{};
     MapVersion mapVersion = VALVE_220; // TODO: Change to MAP_TYPE_QUAKE
 
     // TODO: Sane loading of Maps to be system independent ( see other resource loading ).
 #ifdef _WIN32
-    std::string mapData = loadTextFile(m_ExePath + "../../assets/maps/temple2.map");
+    std::string mapData = loadTextFile(m_ExePath + "../../assets/maps/enemy_test.map");
 #elif __LINUX__
-    std::string mapData = loadTextFile(m_ExePath + "../assets/maps/temple2.map");
+    std::string mapData = loadTextFile(m_ExePath + "../assets/maps/enemy_test.map");
```

Comment:
this map doesn't have any textures yet.

Reply:
Actually, we probably should architect things so that the game never can crash if a resource file could not be found. Let us talk about this tomorrow.